Demo/src/main/java/com/paypal/android/api/services/PayPalDemoApi.kt

```diff
@@ -1,23 +1,20 @@
 package com.paypal.android.api.services
 
 import com.google.gson.JsonObject
+import com.paypal.android.api.model.AccessToken
 import com.paypal.android.api.model.CreateOrderRequest
 import com.paypal.android.api.model.Order
 import retrofit2.http.Body
 import retrofit2.http.POST
-import retrofit2.http.Query
 
 interface PayPalDemoApi {
 
-    @POST("/order")
-    suspend fun fetchOrderId(
-        @Query("countryCode") countryCode: String,
-        @Body orderRequest: CreateOrderRequest?
-    ): Order
+    @POST("/orders")
+    suspend fun fetchOrderId(@Body orderRequest: CreateOrderRequest?): Order
 
-    @POST("/order")
-    suspend fun fetchOrderId(
-        @Query("countryCode") countryCode: String,
-        @Body jsonObject: JsonObject
-    ): Order
+    @POST("/orders")
+    suspend fun fetchOrderId(@Body jsonObject: JsonObject): Order
```

Comment:
`createOrder` for method name instead of `fetchOrderId`? because we have different access and endpoints for get and creating orders
same for above method